### DISCUSSION TITLE
[3.1.16] - Autoselect - get current dropdown

### DISCUSSION THREAD

**Discussion by AniaKru95** [vuetifyjs/vuetify#17236] [Q&A]
Hello,

Is there any way to get current dropdown options from autocomplete?

https://codepen.io/AniaKru95/pen/MWPoeQa

1. Clear "foo"
2. Write only "f"
3. Displayed results "foo, fizz"
(I dont select it, but I want to save it in some variable)

**Comment by yuwu9145** [comment]
You can use `VCombobox` instead https://vuetifyjs.com/en/components/combobox/

[demo link](https://play.vuetifyjs.com/#eNqlU01vnDAQ/SsjX2ilBTfZ24pEqdR/sezBLIPqFn/INrQN4r93bEBh2Rwi5YDMzHvjN/MGziP7bm0x9MhOrAyobCcCPlcaoBxyYW16TcFVuGaJAMYRJLE9TNOaihSjg5AaHbRdL9/YCXTmzyax8Dugxz9V7OGxYjfwQlC1qc3fWwBgyJVpsKO6QXQ9+ortGafUHhHSeY93ok71P7AVfRf2hOeSv6nftp2AbjsbZbbDzYzFidW/mFsNjMFsbck3llPor07aAB5DH2GprHEBRnDYwgStMwoy2lUWySThw7KFp8j4cs5aY7IDZLVw8Wjl62sKezovX9eS2bKlJpUQVPJZm1TZgc3CuRK2+OWNpm9jjI2TmwkgP0+QMjFHDcW4Yj9DsP7E+bXRVEb7kYMrNAaureIvROOu10EqzBujXo7FsfiW1xhEceSN9GGLFuhVXpOrHh3dVbHDRi3I9t8HFBcmCT0WD7PCkspp+z5K3F3NSW9AlzvUDTp0Hx1sV/bOcDvG3YCxCfqXJnK/k/q33xl/9cn089rrpyZPl8WbLlFwOrBtu9RAqmOX//bgTG0=)
